src/app/model/schema.ts

```diff
@@ -208,9 +208,9 @@ export interface MissionInterface {
 
   deliveryWindow: TimeWindow | null;
   deliveryLocation: Location; // default to recipient location
-  deliveryConfirmationImage: ImageUrl;
-  deliveryType: "curbside" | "delivery";
+  deliveryConfirmationImage: any;
   deliveryNotes: string;
+  deliveryType: string;
```

Comment:
"curbside" and "delivery" changes in the scope of this ticket?

Reply:
that was added when I pulled from master and broke everything unless I added line 213 to mission interface